doc/sphinx/language/core/inductive.rst

```diff
@@ -51,10 +51,12 @@ Inductive types
 
    .. exn:: Non strictly positive occurrence of @ident in @type.
 
-      The types of the constructors have to satisfy a *positivity condition*
-      (see Section :ref:`positivity`). This condition ensures the soundness of
-      the inductive definition. The positivity checking can be disabled using
-      the :flag:`Positivity Checking` flag (see :ref:`controlling-typing-flags`).
+      The types of the constructors have to satisfy a *positivity
+      condition* (see Section :ref:`positivity`). This condition
+      ensures the soundness of the inductive definition. The
+      positivity checking can be disabled using the :flag:`Positivity
```

Comment:
```suggestion
      ensures the soundness of the inductive definition. 
      Positivity checking can be disabled using the :flag:`Positivity
```

Reply:
Why should this be uppercase?

Reply:
After removing "the", it's the beginning of a sentence.

Reply:
You should take this suggestion into account IMHO.

Reply:
Thanks for the reminder.